internal/commands/image/rm.go

```diff
@@ -56,6 +58,6 @@ func runRm(bundleStore store.BundleStore, app string) error {
 		return err
 	}
 
-	fmt.Println("Deleted: " + reference.FamiliarString(tagged))
+	_, _ = fmt.Fprintln(dockerCli.Out(), "Deleted:", reference.FamiliarString(tagged))
```

Comment:
this could be
```suggestion
	_, _ = fmt.Fprint(dockerCli.Err(), "Deleted: ")
	_, _ = fmt.Fprintln(dockerCli.Out(), reference.FamiliarString(tagged))
```

But we haven't done that in the CLI (not super useful for `rm`, but it might be worth exploring that approach)

Reply:
Sounds weird to me to split message this way between stderr and stdout. stderr should just be used to report bad conditions, shoudn't it ?

Reply:
No, `stderr` is for informational output. `stdout` for useful output; see https://github.com/docker/cli/pull/1859/files

That way you can either pipe, or capture the `stdout` output, and redirect or suppress `stderr`

e.g. (example taken from that PR)

```bash
docker container run -d nginx:alpine | xargs docker container inspect --format 'the name of the started container is: {{.Name}}'

Unable to find image 'nginx:alpine' locally
alpine: Pulling from library/nginx
cd784148e348: Already exists
6e3058b2db8a: Already exists
7ca4d29669c1: Already exists
a14cf6997716: Already exists
Digest: sha256:385fbcf0f04621981df6c6f1abd896101eb61a439746ee2921b26abc78f45571
Status: Downloaded newer image for nginx:alpine
the name of the started container is: /vigilant_zhukovsky
```

The "pulling" output etc is printed on `stderr`, but the container ID is printed on `stdout`. That way you can still use the container ID to reference for the next command

Reply:
That's how we hacked stderr vs stout, not how it's supposed to be used as a regular posix process.

Reply:
It's at least the convention; given there's no `stdinfo`, we'll have to work with `stdout` and `stderr`. Output such as the above _definitely_ should not be piped to other commands.

(I think I had a link somewhere, but stackoverflow may do for now); `stderr` is _not_ just for errors (or "bad conditions");  https://unix.stackexchange.com/a/336989/78656

Reply:
Any other command you know (outside docker CLI) to do such a use of stderr ?